docs/application/native/guides/internationalization/i18n.md

```diff
@@ -871,9 +871,9 @@ The number-minus-offset value is formatted using a Number Format for the Plural
 
```

Comment:
An unquoted pound sign (#) in the selected sub-message itself (i.e., outside of arguments nested in the sub-message) is replaced by the input number minus the offset. The number-minus-offset value is formatted using a Number Format for the Plural Format's locale.
Please change as:
An unquoted pound signal (#) is a selected sub message (that is, outside of argument nested in the sub message) is replaced by the input number-minus-offset value. The ` number-minus-offset ` value is formatted using a Number format for the PluralFormat locale.

That argument is formatting without subtracting the offset! If you need a custom format and have a non-zero offset, then you need to pass the number-minus-offset value as a separate parameter.
Please change as:
That argument is formatting without subtracting the offset. If you need a custom format and have a non-zero-offset, then you need to pass the ` number-minus-offset ` value as a separate parameter.
Question: When does the argument format without subtracting the offset?